tests/unit/test_client.py

```diff
@@ -127,12 +127,12 @@ def test_quotas_defaults(self):
         TOTAL_SIZE = 67890
         DATA = {
             "quota": {
-                "managedZones": str(MANAGED_ZONES),
-                "resourceRecordsPerRrset": str(RRS_PER_RRSET),
-                "rrsetsPerManagedZone": str(RRSETS_PER_ZONE),
-                "rrsetAdditionsPerChange": str(RRSET_ADDITIONS),
-                "rrsetDeletionsPerChange": str(RRSET_DELETIONS),
-                "totalRrdataSizePerChange": str(TOTAL_SIZE),
+                "managedZones": MANAGED_ZONES,
+                "resourceRecordsPerRrset": RRS_PER_RRSET,
+                "rrsetsPerManagedZone": RRSETS_PER_ZONE,
+                "rrsetAdditionsPerChange": RRSET_ADDITIONS,
+                "rrsetDeletionsPerChange": RRSET_DELETIONS,
+                "totalRrdataSizePerChange": TOTAL_SIZE,
             }
```

Comment:
https://cloud.google.com/dns/docs/reference/v1/projects#resource already returns integers so this conversion seems unnecessary.